db/sql_implementation.go

```diff
@@ -143,6 +153,16 @@ CREATE TABLE IF NOT EXISTS metadata (
 	ethRPCRequestsSentInCurrentUTCDay BIGINT NOT NULL,
 	startOfCurrentUTCDay              DATETIME NOT NULL
 );
+
+CREATE TABLE IF NOT EXISTS peerstore (
+	key  TEXT NOT NULL UNIQUE,
+	data BYTEA NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS dhtstore (
+	key  TEXT NOT NULL UNIQUE,
+	data BYTEA NOT NULL
+);
```

Comment:
As mentioned offline, I haven't been able to find anything about hash vs b-tree indexes in sqlite. Do you know of any documentation that outlines when each type of index is created? 

It would make sense that they'd use a hash index in this case if it is supported.